geoh5py/objects/drape_model.py

```diff
@@ -164,29 +147,106 @@ def prisms(self) -> np.ndarray | None:
         if self._prisms is None and self.on_file:
             self._prisms = self.workspace.fetch_array_attribute(self, "prisms")
 
-        if self._prisms is not None:
-            return np.array(self._prisms.tolist())
+        return np.array(self._prisms.tolist())
 
+    @property
+    def rotation(self):
```

Comment:
I guess there is a good reason for these functions to exist?
As they are not used ghere, could be nice to say why they are here for programmers understanding